src/commands/mobile_app/mod.rs

```diff
@@ -35,6 +36,8 @@ pub fn make_command(mut command: Command) -> Command {
 }
 
 pub fn execute(matches: &ArgMatches) -> Result<()> {
+    debug!("EXPERIMENTAL: The mobile-app subcommand is experimental and may be updated without notice in the future.");
```

Comment:
Debug logs are not shown by default, as the default log level is `WARN`. 

I will update this call to a `warning!` call.